src/Common/tests/Performance/PerfTestBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Base class for performance test classes</summary>
+    public abstract class PerfTestBase
+    {
+        /// <summary>
+        /// Helper method to create a string containing a number of random
+        /// characters equal to the specified length
+        /// </summary>
+        public static string CreateString(int length)
+        {
+            StringBuilder builder = new StringBuilder();
+            StringBuilder empty = new StringBuilder();
+            while (builder.Length < length)
+            {
+                string toAppend = Guid.NewGuid().ToString();
```

Comment:
We could probably just use System.Random.NextInt() and cast it to a char or something here instead of creating GUIDs and calling ToString(). We'd have more control over the random element if we did that. It may be slightly efficient, as well.